README.md

```diff
@@ -140,6 +140,7 @@ Add to your `.babelrc`'s plugins array.
 
 ## Benchmarks
 > Bootstrap: `npm run bootstrap`
+> Build: `npm run build`
```

Comment:
Ah shit, I read bootstrap as benchmark... Speaking of which, how come there isn't an npm alias for benchmark?